bedrock/mozorg/redirects.py

```diff
@@ -246,6 +246,9 @@ def decider(request, **kwargs):
     # to force it to match all repeating slashes in one go.
     redirect(r'^(.*[^/])//+([^/].*)$', '/{}/{}', locale_prefix=False),
 
+    # bug 1237875
+    redirect(r'^community/forums/?$', 'mozorg.about.forums.forums'),
+
     # bug 927442
     redirect(r'^(firefox/)?community/?', 'mozorg.contribute'),
```

Comment:
Note: I had to add the redirect above this line, which was causing the URL to redirect to the `/contribute` page.